src/pages/home/ReportScreen.js

```diff
@@ -583,4 +581,14 @@ export default compose(
         },
         true,
     ),
+    withOnyx(
+        {
+            // This is added down here so that it can access the full report object fetched from Onyx in the previous withOnyx.
+            parentReportActions: {
+                key: ({report}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report ? report.parentReportID : 0}`,
```

Comment:
It shouldn't be possible, no. I checked all of the other places where the parent report actions are fetched using `withOnyx` and they all use the ternary. It's a little inconsistent whether the ternary returns `0` or the string `'0'`, but since it's a template string that will always cast it all to a string anyway, I guess it doesn't really matter.